internal/ui/login/static/i18n/de.yaml

```diff
@@ -80,6 +80,24 @@ MfaInitDone:
   Title: Multifaktor Verifizierung erstellt
   Description: Multifikator Verifizierung erfolgreich abgeschlossen. Der Multifaktor muss bei jeder Anmeldung eingegeben werden, dies beinhaltet auch den aktuellen Authentifizierungs Prozess.
 
+MfaInitU2F:
+  Title: Multifaktor U2F / WebAuthN hinzufügen
```

Comment:
Describe U2F and WebaAuthN?

Reply:
i'd postpone this to later (improving login flow and usability)